src/tooltool/api/tooltool_api/__init__.py

```diff
@@ -19,6 +20,7 @@
 def custom_handle_default_exceptions(e: Exception) -> typing.Tuple[int, str]:
     '''Conform structure of errors as before, to make it work with client (tooltool.py).
     '''
+    import cli_common.log
```

Comment:
If you remove this line, we are ok:
`./tooltool_api/__init__.py:23:5: F401 'cli_common.log' imported but unused`